packages/terra-worklist-data-grid/tests/jest/WorklistDataGrid.test.jsx

```diff
@@ -45,6 +73,24 @@ describe('WorklistDataGrid', () => {
     expect(wrapper).toMatchSnapshot();
   });
 
+  it('test selectable cells', () => {
```

Comment:
I believe it blocks are typically structured to sound like a sentence, so can we change this to something like:
```suggestion
  it('renders a grid with selectable cells', () => {
```